extensions/commands/sbom/cmd_cyclonedx.py

```diff
@@ -129,16 +129,16 @@ def create_component(node: 'Node') -> Component:
                 purl=purl,
                 description=node.conanfile.description
             )
-        if node.conanfile.homepage and cyclonedx_major_version_is_4():  # bug in cyclonedx 3 enforces hashes
+        if node.conanfile.homepage and cyclonedx_major_version_is_5():  # bug in cyclonedx 3 enforces hashes
             component.external_references.add(ExternalReference(
                 type=ExternalReferenceType.WEBSITE,  # noqa
                 url=XsUri(node.conanfile.homepage),
             ))  # noqa
         return component
 
     def me_as_tool() -> Tool:
-        tool = Tool(name="conan extension recipe:create-sbom")
-        if cyclonedx_major_version_is_4():
+        tool = Tool(name="conan extension sbom:cyclonedx")
```

Comment:
this change of the route - is a breaking change, right? 

... even though it was already documented as the "new" value in the README.

Reply:
I am not sure what you mean by "route". I just stumbled across this outdated tool name. Something that I overlooked in #66. Should we keep it as it was?

Reply:
understood.